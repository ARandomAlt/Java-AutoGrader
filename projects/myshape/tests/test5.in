5700
4868