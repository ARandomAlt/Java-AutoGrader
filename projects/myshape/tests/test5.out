Rectangle length: Rectangle width: Rectangle Perimeter: 21136
Rectangle Area: 27747600